### PR TITLE
fix: alembic migration error msg trying to delete constraint on tables

### PR DESCRIPTION
**Pull request by dpgaspar**
### SUMMARY
Alembic migration error msg on PG:

```
INFO  [alembic.runtime.migration] Running upgrade e5ef6828ac4e -> 18532d70ab98, Delete table_name unique constraint in mysql
(psycopg2.errors.UndefinedObject) constraint "table_name" of relation "tables" does not exist
[SQL: ALTER TABLE tables DROP CONSTRAINT table_name]
```

### ADDITIONAL INFORMATION
- [ ] Has associated issue:
- [ ] Changes UI
- [x] Requires DB Migration.
- [ ] Confirm DB Migration upgrade and downgrade tested.
- [ ] Introduces new feature or API
- [ ] Removes existing feature or API

### CODE REVIEW

**Review by villebro — APPROVED** [review]
LGTM. I think this migration shows that we could do with some migration squashing for Superset 1.0, as there's obviously some divergence even between Postgres and MySQL.

**Comment by dpgaspar** [comment]
> LGTM. I think this migration shows that we could do with some migration squashing for Superset 1.0, as there's obviously some divergence even between Postgres and MySQL.

Totally!

File: superset/migrations/versions/18532d70ab98_fix_table_unique_constraint_in_mysql.py [hunk 1]
```diff
@@ -27,16 +27,20 @@
...
-        print(ex)
 
 
 def downgrade():
-    pass
+    bind = op.get_bind()
+    if isinstance(bind.dialect, MySQLDialect):
+        # index only exists in mysql db
+        with op.get_context().autocommit_block():
+            op.add_constraint("table_name", "tables", type_="unique")
```
Comment (bkyryliuk) [inline superset/migrations/versions/18532d70ab98_fix_table_unique_constraint_in_mysql.py:46]: AttributeError: module 'alembic.op' has no attribute 'add_constraint'

I would say - let's pass on the downgrade. That constraint was never needed

**Review by bkyryliuk — APPROVED** [review]
LG%nit

**Pull request merged**